CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - `RiskIndicator` and `RiskSeverity` filter classes to new `py42.sdk.queries.fileevents.filters.risk_filter` module.
 
+### Removed
+
+- Python 2 and 3.5 usage in the library. Only 3.6+ compatible libraries are being used.
```

Comment:
More succinctly and more importantly, I would say `py42 no longer supports python 2.7 or python 3.5`.